src/lib/errors/unsupported-feature-flag-error.ts

```diff
@@ -0,0 +1,11 @@
+import {CustomError} from './custom-error';
+
+export class UnsupportedFeatureFlagError extends CustomError {
+    private static ERROR_CODE: number = 403;
+
+    constructor(featureFlag: string, userMessage = `Feature flag '${featureFlag}' is not enabled for your org`) {
```

Comment:
Maybe different wording like:
`Feature flag '${featureFlag}' is not currently enabled for your org, to enable please contact snyk support`
```suggestion
    constructor(featureFlag: string, userMessage = `Feature flag '${featureFlag}' is not currently enabled for your org, to enable please contact snyk support`) {
```